js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,14 @@
+function lastIndexOf(...args) {
+  if (args.length === 2 && Array.isArray(args[1])) {
+    for (let i = args[1].length - 1; i >= 0; i--) {
+      if (args[1][i] === args[0]) return i;
+    }
+  }
+  return -1;
+}
+
```

Comment:
Code is efficient but you could definitely reduce the chances of typos by using inbuilt methods such as .reverse() and indexOf() and a for...of loop. (I understand that you want to avoid using 'lastIndexOf()' for obvious reasons!)